napi-inl.h

```diff
@@ -998,6 +998,29 @@ inline Symbol Symbol::WellKnown(napi_env env, const std::string& name) {
   return Napi::Env(env).Global().Get("Symbol").As<Object>().Get(name).As<Symbol>();
 }
 
+inline Symbol Symbol::For(napi_env env, const std::string& description) {
```

Comment:
```suggestion
inline Symbol Symbol::For(Napi::Env env, const std::string& description) {
```

... and a similar change for all APIs below.